flixel/input/gamepad/id/MFiID.hx

```diff
@@ -25,8 +25,8 @@ class MFiID
 	public static inline var DPAD_LEFT:Int = 19;
 	public static inline var DPAD_RIGHT:Int = 20;
 	
-	public static var LEFT_ANALOG_STICK(default, null) = new FlxGamepadAnalogStick(0, 1);
-	public static var RIGHT_ANALOG_STICK(default, null) = new FlxGamepadAnalogStick(2, 3);
+	public static var LEFT_ANALOG_STICK (default, null) = new FlxGamepadAnalogStick(0, 1, {up:21, down:22, left:23, right:24});
```

Comment:
The array indices thing is a good argument against my suggestion. I think we might even have had the exact same conversation about those other fake IDs a while ago.. :D